hack/gomakedeps.sh

```diff
@@ -4,16 +4,17 @@
 
 set -e
 
+GO="${GO:-go}"
 target_bin="$1"
 target_pkg="$2"
 shift 2
 fmt=$(cat <<EOF
 {{ if not .Standard }}
 $target_bin: {{\$dir := .Dir}}{{ range .GoFiles }}{{\$dir}}/{{.}} {{end}}
-{{if .Module}}$target_bin: {{.Module.GoMod}}{{end}}
+{{if .Module}}"$target_bin: {{.Module.GoMod}}{{end}}
```

Comment:
I don't understand the single `"`. Did you miss one or was this intentional?

Reply:
Weird. Not sure how that snuck in.